src/overlays/actors/ovl_Obj_Raillift/z_obj_raillift.c

```diff
@@ -1,3 +1,9 @@
+/*
+ * File: z_obj_raillift.c
+ * Overlay: Obj_Raillift
+ * Description: OOT Water Temple Waterfall Platform and Moving Deku Flower Platform
```

Comment:
```suggestion
 * Description: Moving Deku Flower Platform and OOT Water Temple Waterfall Platform
```
I think making the MM usage of this actor the main point, then OoT's usage as the secondary comment would be better